### PR TITLE
Feature/releases page breadcrumbs

### PR DESCRIPTION
**Pull request by andrewpatto**
This is a partial PR for changes to the layout of Release children.
It introduces a new breadcrumb for releases - and new routing mechanisms.
The actual 'data access log' and 'audit log' pages are yet to be correctly refactored.

### CODE REVIEW

File: application/backend/src/app.ts [hunk 1]
```diff
@@ -70,6 +70,8 @@ export class App {
       await this.server.register(fastifyHelmet, {
         contentSecurityPolicy: {
           directives: {
+            // TODO: derive form action hosts from configuration of OIDC
+            formAction: ["'self'", "*.cilogon.org"],
```
Comment (andrewpatto) [inline application/backend/src/app.ts:74]: Just a CSP thing that only came to light when I tried to bounce an OIDC login

File: application/frontend/src/index-router.tsx [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+  const ReleaseBreadcrumbDropdown = () => (
+    <>
+      {releaseChildren.map((c) => (
+        <BreadcrumbDropdownItem to={`../${c.path}`} text={c.text} />
+      ))}
+    </>
+  );
+
+  return createBrowserRouter(
+    createRoutesFromElements(
```
Comment (andrewpatto) [inline application/frontend/src/index-router.tsx:110]: This is the new mechanism for React routing 6.5+ (so called 'data' routing).

In allows each route to have extra data ('handle') that can be used for feeding to breadcrumbs etc

File: application/frontend/src/layouts/layout-base.tsx [hunk 3]
```diff
@@ -87,15 +87,6 @@ export const LayoutBase: React.FC<PropsWithChildren<Props>> = ({
           >
             {loggedInUser ? (
               <ul className="list-reset flex-1 items-center px-4 pb-4 md:px-0 lg:flex lg:pb-0">
-                <li className="my-2 mr-6 md:my-0">
```
Comment (andrewpatto) [inline application/frontend/src/layouts/layout-base.tsx:90]: I finally got rid of the home page that said/did nothing - we start on Releases now

**Comment by andrewpatto** [comment]
Screenshot 2023-03-02 at 11 43 29 am

This is what the breadcrumb looks like - with more 'sibling' pages yet to be added to the dropdown (future PR)

**Review by mmalenic — APPROVED** [review]

**Review by williamputraintan — APPROVED** [review]

**Pull request merged**